<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonDialog</class>
 <widget class="QDialog" name="PersonDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Register</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_firstname">
       <property name="text">
        <string>First name</string>
       </property>
       <property name="buddy">
        <cstring>edit_firstname</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="edit_firstname"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_lastname">
       <property name="text">
        <string>Last name</string>
       </property>
       <property name="buddy">
        <cstring>edit_lastname</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="edit_lastname"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_address">
       <property name="text">
        <string>Address</string>
       </property>
       <property name="buddy">
        <cstring>edit_address</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="edit_address"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_registered">
       <property name="text">
        <string>Registered</string>
       </property>
       <property name="buddy">
        <cstring>edit_registered</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QDateEdit" name="edit_registered"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>PersonDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>PersonDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
